pkg/devicerepository/store/bleve/init.go

```diff
@@ -203,11 +214,58 @@ func (c Config) Initialize(ctx context.Context, lorawanDevicesPath string, overw
 			}
 		}
 
+		// Add the end device provisioning info to the index.
+		endDeviceProfiles, err := s.GetEndDeviceProfilesV2(store.GetEndDeviceProfilesV2Request{BrandID: b.BrandID})
```

Comment:
V2 is not idiomatic TTS. Use `ByXXX` to specialize an existing method name.

Reply:
After checking the comment below, I eventually renamed it to: `GetEndDeviceProfileIdentifiers`.